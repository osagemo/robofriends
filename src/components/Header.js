import React, { PureComponent, Fragment } from "react";
import CounterButton from "./CounterButton";
export default class Header extends PureComponent {
  // shouldComponentUpdate(nextProps, nextState) {
  //   return false;
  // }
  render() {
    return (
      <Fragment>
        <CounterButton color="blue" />
        <h1 className="f1">RoboFriends</h1>
      </Fragment>
    );
  }
}
