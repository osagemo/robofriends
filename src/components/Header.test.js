import { shallow } from "enzyme";
import React from "react";
import Header from "./Header";

it("expect to render Card component", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
