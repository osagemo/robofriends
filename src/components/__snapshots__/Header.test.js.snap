// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<Fragment>
  <CounterButton
    color="blue"
  />
  <h1
    className="f1"
  >
    RoboFriends
  </h1>
</Fragment>
`;
